vignettes/model.Rmd

```diff
@@ -88,68 +88,93 @@ We model the expected number of infections/latent notifications ($\lambda^l$) us
 In some settings there may be additional reporting delays on top of those that are directly observed in the data, and therefore "nowcastable", a common example is the delay from exposure to symptom onset. For these settings we support modelling "latent" reporting delays as a convolution of the underlying expected counts with the potential for these delays to vary over time and by group. This implementation is similar to that implemented in `EpiNow2` and `epidemia` as well as other similar models  [@EpiNow2; @Abbott2020; @Bhatt2020; @Lison2022]. In addition to this we support modelling ascertainment through the use of improper probability mass functions (i.e. by not enforcing a sum to 1 constraint) and inferring ascertainment where possible (for example day of the week reporting patterns).
 
 \begin{align}
-  \lambda_{g,t} &= \nu_{g,t} \sum_{\tau = 0}^{L - 1} F_{g}\left(\tau + 1, t - \tau \right)  \lambda^l_{g, t - \tau} \\
+  \lambda_{g,t} &= \upsilon_{g,t} \sum_{\tau = 0}^{L - 1} F_{g}\left(\tau + 1, t - \tau \right)  \lambda^l_{g, t - \tau} \\
   \nu_{g,t} &= \nu_0 + \beta_{f,\nu} X_{\nu} + \beta_{r,\nu} Z_{\nu}
 \end{align}
 
 Where $\nu_{g,t}$ is the inferred ascertainment and is modelled flexibly using an optional intercept ($\nu_0$), a design matrix ($X_{\nu}$) for fixed effects ($\beta_{f,\nu}$), and a design matrix ($Z_{\nu}$) for random effects ($\beta_{r,\nu}$.
 
 # Delay distribution
 
-Where data is available on the report date for a given individual we can estimate the delay distribution directly and jointly rather than relying on estimates from other sources (though we may wish to augment our priors with these estimates). In the following section we describe our default parametric delay distribution model as well as our highly flexible discrete time to event based generic model.
+Given case counts both by date of reference and by date of report, we can estimate the reporting delay distribution directly and jointly with the nowcast, rather than relying on external estimates from other sources (though we may want to account for external information in our priors). In the following section, we describe our default parametric delay distribution model and its extension into a generic, and its extension into a highly flexible delay model based on discrete time-to-event modelling.
 
 ## Default model
 
-In our default model we consider the delay distribution to follow a discretised log-normal as follows,
+In our default model, we consider the delay to follow a discretised log-normal distribution, specified as
 
 \begin{align}
   p_{g,t,d} &\sim \text{LogNormal} \left(\mu^d, \sigma^d \right) \\
```

Comment:
Actually we might still want to change this here as well for consistency and accuracy (this currently neglects the discretization going on even for the default model). However the disadvantage is that this requires explaining all the discretization here already... Any opinions?

Reply:
I think maybe having a separate section (and possibly/probably) section for discretisation could make sense? I think in future versions it may need quite a bit of fleshing out and there are all the distributions etc to cover? 

The other advantage is that we can reuse it (via linking) if we ever add other parametric dist stuff (i.e if the GT or latent reporting delay stuff moves in model.